open-source/open-source.md

```diff
@@ -55,22 +70,10 @@ Contributing to Open Source projects offers many benefits:
 
 - [Women Who Code](http://womenwhocode.github.io/#project_reviewers) (See 'Featured Repos' and 'How do we contribute to a project?' Sections)
 
-### Lesson
+## Lesson
 
 [Open Source Projects slide deck](https://docs.google.com/presentation/d/13f2I1JbpLNgPcWcAv_HZKKp4-ZeWTcBUDFm-sw2diIk/edit#slide=id.p)
 
-### Common Mistakes / Misconceptions
-
-_"Open source is hard" / "Open source is time-consuming" / "I don't know how to get started" / "I don't have time"_
-
-Getting started on an Open Source project can be intimidating, but it doesn't have to be. A good way to get started is to reference the [slides](https://docs.google.com/presentation/d/13f2I1JbpLNgPcWcAv_HZKKp4-ZeWTcBUDFm-sw2diIk/edit#slide=id.p) on how to get started. Another way is to find something that interests you and see if there is an Open Source project related to it.
-Maybe you are interested in [helping to create a website for food gardeners](https://github.com/Growstuff/growstuff)? Or maybe that now you know how to code, you want to give back in [helping others learn to code](https://github.com/freeCodeCamp/freeCodeCamp)? Or maybe you want to be more specific and [help vets learn how to code](https://github.com/OperationCode/operationcode_frontend)?
-There are tons of projects out there! Once you find one you're interested in, go to the `Issues` section of the repo and [look for a `first-timers-only` label](http://www.firsttimersonly.com/). Not every project will have this label, but if you find an issue with that label, grab it and start working on it!
-
-_"I don't want to code outside of work!"_
-
-If you find an interesting project, your only contribution doesn't have to be just code. You could contribute to documentation, write blog posts, or even answer questions on Stack Overflow.
-
 ### Guided Practice
 
 [Operation Code](https://github.com/OperationCode/operationcode_frontend) could be an interesting place to start with enough set up instructions to get started. The goal of this exercise is to go through the steps of setting up an Open Source project on your computer and get it ready to run locally to be able to start fixing bugs.
```

Comment:
This repo is now-defunct, and the repo that replaced it is really inactive. @daaimah123 do you have a suggestion for a Javascript-based repo I could recommend instead? I do a lot more Python in opensource.

Reply:
```suggestion
The goal of this exercise is to go through the steps of setting up an Open Source project on your computer and get it ready to run locally to be able to start fixing bugs.
```

It looks like there are many other maintained links on the page that I would want them pointed to that don't run the risk of projects becoming inactive. Let's remove this link